packages/modules/viewer-react/src/types.ts

```diff
@@ -131,21 +141,53 @@ export interface ViewerInitializerParams extends ViewerIModelAppOptions {
   /** array of iTwin.js Extensions */
   extensions?: ExtensionProvider[];
 }
-
-export interface ConnectedViewerProps {
-  iTwinId: string;
+export type RequiredViewerProps = XOR<
```

Comment:
maybe we should relook at whats being exported; not a big deal; but everythign here is re-esported at the top level pkgs, and wonder if we should be more aware of what our public api/types are